### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by lxgr-linux**
Added GameMaps for bug fixes and future compatibility with methods needing the name attribute

### CODE REVIEW

**Comment by lxgr-linux** [comment]
Rebased and ready for review.

File: pokete_classes/animations.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import time
 import scrap_engine as se
-
+import pokete_classes.game_map as gm
```
Comment (MaFeLP) [inline pokete_classes/animations.py:6]:
```suggestion


```

File: pokete_classes/game_map.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+"""Map wrapper for compatibility purposes"""
+
+import scrap_engine as se
+
```
Comment (MaFeLP) [inline pokete_classes/game_map.py:4]:
```suggestion


```

File: pokete_classes/game_map.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+class GameMap(se.Map):
+    """Wraps the se.Map class and adds a name attribute
+    ARGS:
+        height: The maps height
+        width: The maps width
+        background: The maps background
+        name: The maps name"""
```
Comment (MaFeLP) [inline pokete_classes/game_map.py:11]:
```suggestion
        height: The map's height
        width: The map's width
        background: The map's background
        name: The map's name"""
```

**Review by MaFeLP — CHANGES_REQUESTED** [review]

File: pokete_classes/animations.py [hunk 4]
```diff
@@ -2,8 +2,7 @@
 
 import time
 import scrap_engine as se
-
-
+import pokete_classes.game_map as gm
 def transition(_map, poke):
```
Comment (MaFeLP) [inline pokete_classes/animations.py:6]:
```suggestion
import pokete_classes.game_map as gm


def transition(_map, poke):
```

**Review by MaFeLP — CHANGES_REQUESTED** [review]

**Pull request merged**